app/models/user.rb

```diff
@@ -1,8 +1,10 @@
 # Huginn is designed to be a multi-User system.  Users have many Agents (and Events created by those Agents).
 class User < ActiveRecord::Base
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable, :lockable,
-         :omniauthable
+  DEVISE_MODULES = [:database_authenticatable, :registerable,
+                    :recoverable, :rememberable, :trackable,
+                    :validatable, :lockable, :omniauthable,
+                    (ENV['ALLOW_UNCONFIRMED_ACCESS'] == 'false' ? :confirmable : nil)].compact
```

Comment:
Very good point, if found that a little confusing myself, of course just inverting the logic did not came to mind :smile:. I went with `REQUIRE_CONFIRMED_EMAIL` though, does that work for you?